### PR TITLE
[#6814] wagtail v4 LTS update

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

File: euth_wagtail/settings/base.py [hunk 1]
```diff
@@ -365,6 +365,10 @@
 
 WAGTAIL_SITE_NAME = "euth_wagtail"
 
+# Base URL to use when referring to full URLs within the Wagtail admin backend -
+# e.g. in notification emails. Don't include '/admin' or a trailing slash
+WAGTAILADMIN_BASE_URL = "http://localhost:8000"
+
```
Comment (philli-m) [inline euth_wagtail/settings/base.py:371]: @goapunk I added this and wasn't sure if it also meant we would have to overwrite in our pillar but I assume not?

Comment (fuzzylogic2000) [reply euth_wagtail/settings/base.py:371]: Yes, we do! Will add that as a task.

**Comment by fuzzylogic2000** [comment]
Sorry, I merged some updates and produced conflicts. 
Is this ready to review apart from that?

File: renovate.json [hunk 2]
```diff
@@ -28,6 +28,10 @@
       {
         "matchPackageNames": ["Django"],
         "allowedVersions": "=3.2"
+      },
+      {
+        "matchPackageNames": ["wagtail"],
+        "allowedVersions": "<4.2"
```
Comment (fuzzylogic2000) [inline renovate.json:34]: I think we decided to go with the `=` as above, so it should be `=4.1`. But also that's not suuperimportant to me.

**Review by fuzzylogic2000 — APPROVED** [review]
Looks good. One comment, but just merge if you don't care. :)

**Pull request merged**